### ISSUE TITLE
NVDA 2023.1: Key labels are displayed in the system language instead of NVDA language set in preferences.

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#14657]
### Steps to reproduce:
1. Set  NVDA to the language different than your Windows language (in my case since my Windows is in English I'm going to use Polish for the rest of this example).
2. After restarting NVDA enable keyboard help and try pressing keys for which translation for your language provides localized labels e.g. space, left and right shift etc.
### Actual behavior:
Key labels are reported in the language of the system i.e. in my case 'space' is reported for the space bar key instead of the Polish equivalent 'spacja'.
### Expected behavior:
Key labels should be reported in the  language set in preferences similar to how this worked in NVDA 2022.4 and earlier.
### NVDA logs, crash dumps and other attachments:
None needed
#### NVDA installed/portable/running from source:
Tested from sources, but this does not matter.
#### NVDA version:
Latest beta
#### Windows version:
Windows 10 version 21H2, but this does not depend on the version of Windows which is used.
#### Name and version of other software in use when reproducing the issue:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
This was introduced in PR #14528, so all versions after this PR have this bug.
#### If NVDA add-ons are disabled, is your problem still occurring?
Not relevant
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not relevant

## Additional details:
Upcoming PR
